spec/griddler/email_spec.rb

```diff
@@ -139,6 +139,20 @@
     expect(body_from_email(text: body)).to eq 'Hello.'
   end
 
+  it 'handles "*From:* email@email.com" format' do
+    body = <<-EOF
+      Hello.
+
+      *From:* bob@example.com
+      *Sent:* Today
+      *Subject:* Awesome report.
+
+      Check out this report!
+    EOF
+
+    expect(body_from_email(text: body)).to eq 'Hello.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.